### ISSUE TITLE
Look into optimizing moment JS locale data

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-toolkit#193]
# Feature Request

## Description
Looking into our usage and evaluate how much of moment.js we pull in. It looks like there may be some savings we could gain with some webpack config updates.

https://github.com/jmblog/how-to-optimize-momentjs-with-webpack

**Comment by emilyrohrbough** [comment]
I wonder how this would fit into our current locale testing strategy.?

**Comment by emilyrohrbough** [comment]
I don't think this changes should be made in our default configuration since lot of projects will not use moment. I think the best approach would be for teams to merge our default webpack configuraiton with any desired changes they need.
